indico/cli/devserver.py

```diff
@@ -32,6 +32,9 @@ def run_watchfiles():
 
 
 def run_server(info, host, port, url, ssl, ssl_key, ssl_cert, quiet, proxy, enable_evalex, evalex_from):
+    if ':' in host and host.count(':') > 2:
+        raise ValueError('<your-hostname> should not contain port number. Pass port number using -p argument.')
```

Comment:
That's oddly specific. It would be nicer to add a validation callback here:

```
@click.option('--host', '-h', default='127.0.0.1', metavar='HOST', help='The ip/host to bind to.')
```

Reply:
Added specific check for these kind of addresses ::1:9000 which are not handled currently

The address if passed in a format like 127.0.0.1:9000 is handled currently as it throws 
`ValueError: '127.0.0.1:9000' does not appear to be an IPv4 or IPv6 address`